### ISSUE TITLE
Try to use Qt 5.15.2 for Windows

### ISSUE THREAD

**Issue by pbek** [pbek/QOwnNotes#1976]
Due to bugs in windeployqt the build process for Windows was pinned to Qt 5.13.
This is an attempt to use Qt 5.15.2 for Windows too.

**Comment by pbek** [comment]
windeployqt says:

```
    Directory: D:\a\QOwnNotes\QOwnNotes

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----           12/6/2020  9:23 AM                release
D:\a\QOwnNotes\QOwnNotes\release\QOwnNotes.exe 64 bit, release executable [QML]
Unable to find the platform plugin.
Direct dependencies: Qt5Core Qt5Gui Qt5Network Qt5PrintSupport Qt5Qml Qt5Sql Qt5WebSockets Qt5Widgets Qt5Xml Qt5XmlPatterns
All dependencies   : Qt5Core Qt5Gui Qt5Network Qt5PrintSupport Qt5Qml Qt5Sql Qt5WebSockets Qt5Widgets Qt5Xml Qt5XmlPatterns
To be deployed     : Qt5Core Qt5Gui Qt5Network Qt5PrintSupport Qt5Qml Qt5Sql Qt5WebSockets Qt5Widgets Qt5Xml Qt5XmlPatterns

Error: Process completed with exit code 1.
```

**Comment by pbek** [comment]
I finally did it!

## 20.12.3
- the Windows releases of QOwnNotes are now built with Qt 5.15.2

**Comment by pbek** [comment]
There are now also release builds as build artefact for every git commit.

**Comment by alexanderino** [comment]
Thanks. In Windows, I removed the following (found in 20.12.2 but not in 20.12.3):

- Qt5RemoteObjects.dll [file]
- QtQml [directory]
- QtWebSockets [directory]

Have fired up 20.12.3, running fine so far.

**Comment by pbek** [comment]
Thank you for the feedback!

> Thanks. In Windows, I removed the following (found in 20.12.2 but not in 20.12.3):

What happened before you removed them?
Where those distributed by a previous version of QOwnNotes?
I had no issues updating (and at least one other person didn't seem to have troubles too)...

**Comment by alexanderino** [comment]
> What happened before you removed them?

Nothing. I identified and removed them before running `20.12.3`.

> Where those distributed by a previous version of QOwnNotes?

Yes. Since I began using QOwnNotes (since August/September 2020?), there had always been 100 objects in the zip file's root directory. `20.12.3` is the first with a different count: 98. I would imagine it has to do with the new version of Qt not needing those files.

I can confirm that there are no file or directory name differences between `20.9.11` and `20.12.2`.

> I had no issues updating (and at least one other person didn't seem to have troubles too)...

I don't think there's any harm in leaving them there — at least for now :smiley:
